### ISSUE TITLE
Publish Gitea binaries on ghcr as well

### ISSUE THREAD

**Issue by dasfuu** [go-gitea/gitea#22922]
### Feature Description

Since the docker hub is rate limited it would be very nice to provide the official container images on some alternative registries like github.

**Comment by jolheiser** [comment]
I'm assuming you are referring to adding the docker images for running Gitea to ghcr. 

If you mean supporting a feature like ghcr in Gitea, that is already available.

**Comment by dasfuu** [comment]
Yes, i meant adding the docker images for running Giteo to ghcr.

**Comment by james7132** [comment]
Seconding this issue, Docker (the company) is retiring their free tier before the end of this month: https://blog.alexellis.io/docker-is-deleting-open-source-images/. It'd be ideal for people to have a migration path, or just simply an alternative in the case DockerHub goes under.

**Comment by chris-sanders** [comment]
Running into rate limiting on docker.io would be nice to have a ghcr.io to pull from instead.

**Comment by qwe1231adsd** [comment]
This issue became a hot topic after Docker Hub was blocked in sub-sanctioned countries in the US. Could you use Github Artifact repository? I saw that it is present in Google(mirror.gcr.io/gitea/gitea:1.22.0), but for some reason it is missing in Github

**Comment by TheFox0x7** [comment]
resolved in https://github.com/go-gitea/gitea/pull/34148
